test/graph_gen_test.py

```diff
@@ -135,14 +105,8 @@
 76
 >>> testfile_3_p['graph']['edges'][1]['weight']
 15
->>> testfile_3_p['properties']['signed']
-False
 >>> testfile_3_p['properties']['directed']
 True
->>> testfile_3_p['properties']['self_loop']
-True
->>> testfile_3_p['properties']['multigraph']
```

Comment:
Please revert all `multigraph` tests!

Reply:
Sorry for that; it was fixed in the new commit.